cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/operationrunner/InMemorySourceControlOperationRunner.java

```diff
@@ -76,7 +86,8 @@ public PushAppResponse push(PushAppContext pushContext) throws PushFailureExcept
   }
 
   @Override
-  public PullAppResponse<?> pull(ApplicationReference appRef, RepositoryConfig repoConfig) throws PullFailureException,
+  public PullAppResponse<?> pull(ApplicationReference appRef, RepositoryConfig repoConfig)
+    throws SourceControlException,
```

Comment:
nit: can remove `SourceControlException` and `IllegalArgumentException`

Reply:
Done